src/ansys/mapdl/core/plotting.py

```diff
@@ -625,6 +646,13 @@ def general_plotter(
         Size of the text on the boundary conditions labels.
         By default it is 16.
 
+    plotter : pyvista.Plotter, optional
+        If a pyvista.Plotter not is provided, then creates its
+        own plotter. If a pyvista.Plotter is provided, the arguments
+        ``notebook``, ``off_screen`` and ``theme`` are ignored, since
+        they should be set when instantiated the provided plotter.
+        Defaults to None (create the Plotter object)
```

Comment:
```suggestion
        If a :class:`pyvista.Plotter` not is provided, then creates its
        own plotter. If a :class:`pyvista.Plotter` is provided, the arguments
        ``notebook``, ``off_screen`` and ``theme`` are ignored, since
        they should be set when instantiated the provided plotter.
        Defaults to ``None`` (create the Plotter object).
```